flux-refineck/src/checker.rs

```diff
@@ -1199,9 +1193,9 @@ pub(crate) mod errors {
             Self { kind: CheckerErrKind::OpaqueStruct(def_id), span }
         }
 
-        pub fn query(query_error: QueryErr, span: Span) -> Self {
-            Self { kind: CheckerErrKind::Query(query_error), span }
-        }
+        // pub fn query(query_error: QueryErr, span: Span) -> Self {
+        //     Self { kind: CheckerErrKind::Query(query_error), span }
+        // }
```

Comment:
I think we can get rid of this

```suggestion
        // pub fn query(query_error: QueryErr, span: Span) -> Self {
        //     Self { kind: CheckerErrKind::Query(query_error), span }
        // }
```